docs/release-notes/cloud/cloud-2022-04.md

```diff
@@ -0,0 +1,77 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+description: Release notes for Codacy Cloud April 2022.
+included_jira_versions: ['2022.Q2.2', '2022.Q2.3']
+codacy_tools_version_old: https://github.com/codacy/codacy-tools/releases/tag/5.6.3
+codacy_tools_version_new: https://github.com/codacy/codacy-tools/releases/tag/5.7.31
+---
+
+# Cloud April 2022
+
+These release notes are for the Codacy Cloud updates during April 2022.
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+## Product enhancements
+
+-   Added support for the RuboCop extension [<span class="skip-vale">rubocop-thread_safety</span>](https://github.com/covermymeds/rubocop-thread_safety){: target="_blank"} to check your Ruby code for thread-safety issues. (CY-5991)
+-   Added support for the ESLint plugin [<span class="skip-vale">eslint-plugin-es</span>](https://github.com/mysticatea/eslint-plugin-es){: target="_blank"} to disallow the syntax of specific ECMAScript versions. (CY-5968)
+
+## Bug fixes
+
+-   Fixed a broken link in the notification emails sent to organization admins when new members ask to join an organization. (CY-5979)
+-   Now, the PMD CPD duplication tool runs without the `--ignore-identifiers` flag because it was found to report more false positives. (CY-5955)
+-   Fixed an issue that prevented Codacy from listing GitHub repositories on the Repositories list. (CY-5935)
+-   Fixed an issue that could cause the **Organization Overview** page to display pull requests with the wrong status under the **Most problematic** open pull requests tab. (CY-5872)
```

Comment:
Check if this issue is correctly solved in production, see [my comment on the Jira issue](https://codacy.atlassian.net/browse/CY-5872?focusedCommentId=50029).

Reply:
The issue I noticed has been solved by re-analyzing the pull request. :+1: